mailpoet/mailpoet
# Feat/allow all emails from verified domains to be used for sending - MAILPOET-4601

## Description

In this PR, we made an update to our sending process.

Now, plugin users can send a Newsletter campaign from any email address of their verified domain.


## QA notes

Veljko to Alex notes:
2 domains are already authorized for testing on our test sites: `qawp.net` and `moonbridge.rs`
If you test positive scenario in this pr, you should use any email associated with above 2 domains, no errors/problems with sending etc.
If you test negative scenario in this pr, you should use any other email address not associated with above 2 domains. You should see warnings/errors in the settings page if you try to apply it for Default Sender's From field and/or on the last page when creating and sending newsletter (there's also From field there).


## Linked tickets

[MAILPOET-4601](https://mailpoet.atlassian.net/browse/MAILPOET-4601)
